### ISSUE TITLE
Version 1.27.1 won't start

### ISSUE THREAD

**Issue by MMhunter** [Microsoft/vscode#58163]
- VSCode Version: 1.27.1
- OS Version: Windows 7 SP1 (Internal Version 7061)

Steps to Reproduce:

1. Download and Install VS Code v1.27.1
2. Double click the shortcut.
3. Nothing Happens.

Launch With 'code --verbose':
No output
![image](https://user-images.githubusercontent.com/1378130/45205253-33570b00-b2b4-11e8-8d9a-a8087a0ed592.png)

Does this issue occur when all extensions are disabled?: No

Version 1.26.x works fine.

**Comment by bpasero** [comment]
@MMhunter very weird indeed. I wonder if that installation is just broken. 
Can you try to reproduce with our nightly insider builds? You can give our preview releases a try from: https://code.visualstudio.com/insiders/

**Comment by DeviousM** [comment]
I have the same issue on macOS 10.13.6. Can't open VSCode until I downgrade.

**Comment by DeviousM** [comment]
@bpasero - is something going on regarding this issue? I mean I've provided more information and I guess that is what is needed to go on with it.

**Comment by bpasero** [comment]
@DeviousM does it work if you run "code --disable-gpu" from the command line?

**Comment by DeviousM** [comment]
@bpasero - 1.27.1 wouldn't start with any option from the --help. After updating to recovery build 1.27.2 it started to work so I guess that issue has been resolved. @MMhunter - does your work after updating?

**Comment by bpasero** [comment]
@DeviousM then it sounds like a broken install to me.

**Comment by DeviousM** [comment]
@bpasero - believe me, it was not a broken installation. I've reinstalled the .app file few times just to make sure. Then I've downgraded and it worked.

**Comment by bpasero** [comment]
@MMhunter ping

**Comment by MMhunter** [comment]
@bpasero yep it works now...but in fact the older version works too on the users' machine.  Our app with electron 2.0.8 works too.. Wired indeed. It just recovered somehow.
